flang/LAPACK-bugs.txt

```diff
@@ -13,15 +15,15 @@ ______________
   Intrinsics lowering problems
   . bbc: Lower/Intrinsics.cpp:763: Assertion `false && "LEN_TRIM TODO"' failed.
 
-[unassigned]
+  lapack/BLAS/SRC/zhemm.f:304:27: 'fir.convert' op invalid type conversion
+  . bbc: This looks like the intrinsic lowering is not converting between floats and complex quite right.
```

Comment:
OK, could be, I'll have a look. Intrinsic lowering should not be dealing with such conversions that should be rewritten to Fortran::evaluate::ComplexPart in the front-end. But that is a lot of should :)